### DISCUSSION TITLE
Problem att ladda ner Robinson Gränslandet (DRM-skydd?)

### DISCUSSION THREAD

**Discussion by SweWolf** [stefansundin/privatkopiera#230] [Q&A]
Jag lyckas inte ladda ner Robinson Gränslandet.
Jag ser att flera har nämnt något om ett DRM-skydd på TV4, så kanske att det är det som strular.
Medan jag laddar ner så får jag meddelanden som:  Failed to seek for auxiliary info, will only parse senc atoms for encryption info

ffmpeg -i "https://vod.streaming.a2d.tv/5b392694-6872-4661-862b-2dad78581084/5073af00-71b6-11ef-9af3-896bc81e1e3b_20850861.ism/5073af00-71b6-11ef-9af3-896bc81e1e3b_20850861-video=6500000.m3u8" -i "https://vod.streaming.a2d.tv/5b392694-6872-4661-862b-2dad78581084/5073af00-71b6-11ef-9af3-896bc81e1e3b_20850861.ism/5073af00-71b6-11ef-9af3-896bc81e1e3b_20850861-audio=128000.m3u8" -c:v copy -c:a copy -bsf:a aac_adtstoasc "Gränslandet.mp4"

**Comment by martinbenny** [comment]
Min gissning är att TV4 kommer att börja kryptera alla innehåll. Kan man lösa det med hjälp av cookie från Chrome?

**Comment by BacchusFLT** [comment]
Om jag fattar det rätt så är vissa delar öppet och vissa delar bakom
betalvägg.

Den senare delen kan du som har abonnemang ladda ner med svtplay-dl om du
anger inloggningsuppgifter i kommandot eller definierar dom i YAML filen.
Privatkopiera kan väl inte hantera dessa tror jag.

Sedan är vissa filer DRM skyddade. Det är alltså inte samma som
betalväggsbegränsningen. Dessa funkar inte alls. Varken med Privatkopiera
eller med svtplay-dl.

Den ons 30 okt. 2024 11:07martinbenny ***@***.***> skrev:

**Reply by SweWolf** [reply]
Nja, alltså det finns en gratisdel av TV4 Play, men även den som är gratis kräver användarnamn/lösenord.
Så finns det även betalpaket, t.ex. Plus med reklam, och Plus utan reklam.

**Comment by Bub0** [comment]
DRM.

M3u8 säger,

#EXTM3U
#EXT-X-VERSION:6
## Created with Unified Streaming Platform (version=1.11.23-28141)
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-MEDIA-SEQUENCE:1
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-TARGETDURATION:4
#USP-X-TIMESTAMP-MAP:MPEGTS=900000,LOCAL=1970-01-01T00:00:00Z
#EXT-X-MAP:URI="hls/5073af00-71b6-11ef-9af3-896bc81e1e3b_20850861-video=6500000.m4s"
#EXT-X-PROGRAM-DATE-TIME:1970-01-01T00:00:00Z
#EXT-X-KEY:METHOD=SAMPLE-AES,URI="data:text/plain;base64,AAAAYnBzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAAEISEEDeq2dILDqJh7lCRbTibq4iKGV5SmhjM05sZEVsa0lqb2lkSFp0WldScFlTMHlNRGcxTURnMk1TSjlI88aJmwY=",KEYFORMAT="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed",KEYFORMATVERSIONS="1"
#EXT-X-KEY:METHOD=SAMPLE-AES,URI="data:text/plain;charset=UTF-16;base64,MgMAAAEAAQAoAzwAVwBSAE0ASABFAEEARABFAFIAIAB4AG0AbABuAHMAPQAiAGgAdAB0AHAAOgAvAC8AcwBjAGgAZQBtAGEAcwAuAG0AaQBjAHIAbwBzAG8AZgB0AC4AYwBvAG0ALwBEAFIATQAvADIAMAAwADcALwAwADMALwBQAGwAYQB5AFIAZQBhAGQAeQBIAGUAYQBkAGUAcgAiACAAdgBlAHIAcwBpAG8AbgA9ACIANAAuADMALgAwAC4AMAAiAD4APABEAEEAVABBAD4APABQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsASQBEAFMAPgA8AEsASQBEACAAQQBMAEcASQBEAD0AIgBBAEUAUwBDAEIAQwAiACAAVgBBAEwAVQBFAD0AIgBaADYAdgBlAFEAQwB4AEkAaQBUAHEASAB1AFUASgBGAHQATwBKAHUAcgBnAD0APQAiAD4APAAvAEsASQBEAD4APAAvAEsASQBEAFMAPgA8AC8AUABSAE8AVABFAEMAVABJAE4ARgBPAD4APABMAEEAXwBVAFIATAA+AGgAdAB0AHAAcwA6AC8ALwBsAGkAYwAuAGQAcgBtAHQAbwBkAGEAeQAuAGMAbwBtAC8AbABpAGMAZQBuAHMAZQAtAHAAcgBvAHgAeQAtAGgAZQBhAGQAZQByAGEAdQB0AGgALwBkAHIAbQB0AG8AZABhAHkALwBSAGkAZwBoAHQAcwBNAGEAbgBhAGcAZQByAC4AYQBzAG0AeAA8AC8ATABBAF8AVQBSAEwAPgA8AEwAVQBJAF8AVQBSAEwAPgBoAHQAdABwAHMAOgAvAC8AcABsAGEAeQByAGUAYQBkAHkALQB1AGkALgBlAHgAYQBtAHAAbABlAC4AYwBvAG0APAAvAEwAVQBJAF8AVQBSAEwAPgA8AEQARQBDAFIAWQBQAFQATwBSAFMARQBUAFUAUAA+AE8ATgBEAEUATQBBAE4ARAA8AC8ARABFAEMAUgBZAFAAVABPAFIAUwBFAFQAVQBQAD4APAAvAEQAQQBUAEEAPgA8AC8AVwBSAE0ASABFAEEARABFAFIAPgA=",KEYFORMAT="com.microsoft.playready",KEYFORMATVERSIONS="1"
#EXT-X-KEY:METHOD=SAMPLE-AES,URI="skd://drmtoday?assetId=tvmedia-20850861&variantId&keyId=40deab67482c3a8987b94245b4e26eae",KEYFORMAT="com.apple.streamingkeydelivery",KEYFORMATVERSIONS="1"

**Comment by Lunarwalker60** [comment]
Inget i den lagliga domänen kommer att fixa DRM-problemen.--Sent from my Android phone with mail.com Mail. Please excuse my brevity.On 2024-10-30, 22:20 Bub0 ***@***.***> wrote:

   
  DRM. 
  M3u8 säger, 
  #EXTM3U #EXT-X-VERSION:6 
  Created with Unified Streaming Platform (version=1.11.23-28141) 
  #EXT-X-PLAYLIST-TYPE:VOD #EXT-X-MEDIA-SEQUENCE:1 #EXT-X-INDEPENDENT-SEGMENTS #EXT-X-TARGETDURATION:4 #USP-X-TIMESTAMP-MAP:MPEGTS=900000,LOCAL=1970-01-01T00:00:00Z #EXT-X-MAP:URI="hls/5073af00-71b6-11ef-9af3-896bc81e1e3b_20850861-video=6500000.m4s" #EXT-X-PROGRAM-DATE-TIME:1970-01-01T00:00:00Z #EXT-X-KEY:METHOD=SAMPLE-AES,URI="data:text/plain;base64,AAAAYnBzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAAEISEEDeq2dILDqJh7lCRbTibq4iKGV5SmhjM05sZEVsa0lqb2lkSFp0WldScFlTMHlNRGcxTURnMk1TSjlI88aJmwY=",KEYFORMAT="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed",KEYFORMATVERSIONS="1" #EXT-X-KEY:METHOD=SAMPLE-AES,URI="data:text/plain;charset=UTF-16;base64,MgMAAAEAAQAoAzwAVwBSAE0ASABFAEEARABFAFIAIAB4AG0AbABuAHMAPQAiAGgAdAB0AHAAOgAvAC8AcwBjAGgAZQBtAGEAcwAuAG0AaQBjAHIAbwBzAG8AZgB0AC4AYwBvAG0ALwBEAFIATQAvADIAMAAwADcALwAwADMALwBQAGwAYQB5AFIAZQBhAGQAeQBIAGUAYQBkAGUAcgAiACAAdgBlAHIAcwBpAG8AbgA9ACIANAAuADMALgAwAC4AMAAiAD4APABEAEEAVABBAD4APABQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsASQBEAFMAPgA8AEsASQBEACAAQQBMAEcASQBEAD0AIgBBAEUAUwBDAEIAQwAiACAAVgBBAEwAVQBFAD0AIgBaADYAdgBlAFEAQwB4AEkAaQBUAHEASAB1AFUASgBGAHQATwBKAHUAcgBnAD0APQAiAD4APAAvAEsASQBEAD4APAAvAEsASQBEAFMAPgA8AC8AUABSAE8AVABFAEMAVABJAE4ARgBPAD4APABMAEEAXwBVAFIATAA+AGgAdAB0AHAAcwA6AC8ALwBsAGkAYwAuAGQAcgBtAHQAbwBkAGEAeQAuAGMAbwBtAC8AbABpAGMAZQBuAHMAZQAtAHAAcgBvAHgAeQAtAGgAZQBhAGQAZQByAGEAdQB0AGgALwBkAHIAbQB0AG8AZABhAHkALwBSAGkAZwBoAHQAcwBNAGEAbgBhAGcAZQByAC4AYQBzAG0AeAA8AC8ATABBAF8AVQBSAEwAPgA8AEwAVQBJAF8AVQBSAEwAPgBoAHQAdABwAHMAOgAvAC8AcABsAGEAeQByAGUAYQBkAHkALQB1AGkALgBlAHgAYQBtAHAAbABlAC4AYwBvAG0APAAvAEwAVQBJAF8AVQBSAEwAPgA8AEQARQBDAFIAWQBQAFQATwBSAFMARQBUAFUAUAA+AE8ATgBEAEUATQBBAE4ARAA8AC8ARABFAEMAUgBZAFAAVABPAFIAUwBFAFQAVQBQAD4APAAvAEQAQQBUAEEAPgA8AC8AVwBSAE0ASABFAEEARABFAFIAPgA=",KEYFORMAT="com.microsoft.playready",KEYFORMATVERSIONS="1" #EXT-X-KEY:METHOD=SAMPLE-AES,URI="skd://drmtoday?assetId=tvmedia-20850861&variantId&keyId=40deab67482c3a8987b94245b4e26eae",KEYFORMAT="com.apple.streamingkeydelivery",KEYFORMATVERSIONS="1"
